apache/metron
# METRON-1913: metron-alert UI - Build broken by missing transitive dependency

## Contributor Comments

Metron Alert UI build broken by a removed transitive dependency. flatmap-steam was removed from npm because of vulnerability issues probably.
Bumping version of npm-run-all from 4.1.3 to 4.1.5 fixing the issue. 

WAY TO TEST:
- check out current mater and run: npm ls flatmap-stream
- it's gonna show you that it is a dependency of npm-run-all
- run npm cache clean --force (to remove cached versions of the missing flatmap-stream)
- run npm ci (it should fail)
- checkout this pr branch
- run npm ci (should pass)
- rerun npm ls flatmap-stream
- you should see that it's no longer a dependency

## Pull Request Checklist

Thank you for submitting a contribution to Apache Metron.  
Please refer to our [Development Guidelines](https://cwiki.apache.org/confluence/pages/viewpage.action?pageId=61332235) for the complete guide to follow for contributions.  
Please refer also to our [Build Verification Guidelines](https://cwiki.apache.org/confluence/display/METRON/Verifying+Builds?show-miniview) for complete smoke testing guides.  


In order to streamline the review of the contribution we ask you follow these guidelines and ask you to double check the following:

### For all changes:


### For code changes:
  ```
  mvn -q clean integration-test install && dev-utilities/build-utils/verify_licenses.sh 
  ```


### For documentation related changes:

  ```
  cd site-book
  mvn site
  ```

#### Note:
Please ensure that once the PR is submitted, you check travis-ci for build issues and submit an update to your PR as soon as possible.
It is also recommended that [travis-ci](https://travis-ci.org) is set up for your personal repository such that your branches are built there before submitting a pull request.
